src/main/java/uk/gov/companieshouse/pscfiling/api/validator/CeasedOnDateValidator.java

```diff
@@ -20,12 +20,13 @@ public void validate(final FilingValidationContext validationContext) {
         final PscApi pscDetails = pscDetailsService.getPscDetails(validationContext.getTransaction(),
                 validationContext.getDto().getReferencePscId(), validationContext.getPscType(),
                 validationContext.getPassthroughHeader());
-
         final var ceasedOn = validationContext.getDto().getCeasedOn();
-        if (ceasedOn != null && ceasedOn.isBefore(pscDetails.getNotifiedOn())) {
+        final var notifiedOn = pscDetails.getNotifiedOn();
```

Comment:
Does it matter if the notified_on date is retrieved via the pscDetails rather than the validationContext? Don't think we need the PscDetailsService if we use the validationContext.

Reply:
The `notified_on` comes from the `company_pscs` API, so in turn accessed from the `pscDetails` yeah?